README.md

```diff
@@ -466,9 +466,9 @@ let additions =
           , "effect"
           ]
       , repo =
-          "https://github.com/Unisay/purescript-facebook.git"
+          "https://github.com/Unisay/purescript-facebook.git" -- must be https not ssh
```

Comment:
```suggestion
          "https://github.com/Unisay/purescript-facebook.git"
```

I'd like to skip this comment because there's a pretty comprehensive explanation to convert the URL in the error message you get if you try to use the ssh url, as detailed in https://github.com/spacchetti/spago/issues/348#issuecomment-516712325.
E.g. following the instructions there you could convert this URL to ssh as `ssh://git@github.com/Unisay/purescript-facebook.git`